spec/i18n_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'i18n/tasks'
+
+RSpec.describe 'I18n' do
+  let(:i18n) { I18n::Tasks::BaseTask.new }
+  let(:missing_keys) { i18n.missing_keys }
+  let(:unused_keys) { i18n.unused_keys }
+
+  it 'does not have missing keys' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.